homeassistant/components/vallox/sensor.py

```diff
@@ -107,7 +107,9 @@ def native_value(self) -> StateType | datetime:
         days_remaining_delta = timedelta(days=days_remaining)
         now = datetime.utcnow().replace(hour=13, minute=0, second=0, microsecond=0)
 
-        return now + days_remaining_delta
+        return datetime.fromtimestamp(
+            (now + days_remaining_delta).timestamp(), tz=timezone.utc
+        )
```

Comment:
Why not?

`return (now + days_remaining_delta).astimezone(dt_util.UTC)`

As in,  I'm not completely sure why the juggling with the timestamps is done here?

Reply:
That way is better. I didn't know that exists.